src/webgpu/api/validation/resource_usages/texture/in_pass_encoder.spec.ts

```diff
@@ -255,6 +253,13 @@ const SLICE_COUNT = 2;
 
 // For all tests below, we test compute pass if 'compute' is true, and test render pass otherwise.
```

Comment:
This explanation should be moved into the test descriptions instead of being a code comment. Actually, it's almost self-documenting, so with tweaks to the test descriptions, I think this can just be deleted.

Reply:
ok, removed.